pages/locators/locators.py

```diff
@@ -1,34 +1,64 @@
+    SEARCH_FLIGHT_BUTTON = (By.XPATH, "//button[@data-ref='flight-search-widget__cta']")
+    CONFIRMATION_AIRPORT = (By.XPATH, "(//span[@data-ref='airport-item__name'])[last()]")
+    RETURN_FORM = (By.XPATH, "//fsw-input-button[@uniqueid='dates-to']")
+
+
+class HotelsTabMainPageLocators:
+    SEARCH_HOTEL_BUTTON = (By.XPATH, "//button[@data-ref='rooms-search-widget__cta']")
+    DESTINATION_HOTEL_FORM = (By.XPATH, "//*[@id='input-button__locations-or-properties']")
+    CHECK_IN_FORM = (By.XPATH, "//hp-input-button[@uniqueid='check-in']")
+    CHECK_OUT_FORM = (By.XPATH, "//hp-input-button[@uniqueid='check-out']")
+    CONFIRMATION_HOTEL = (By.XPATH, "//icon[@iconid='glyphs/destination-pin']")
+
+
+class UserMainPageLocators:
     ACCEPT_COOKIES_BUTTON = (By.XPATH,
-                             "//div[@id='cookie-popup-with-overlay']//button[@class='cookie-popup-with-overlay__button']")
+                             "//div[@id='cookie-popup-with-overlay']//button["
+                             "@class='cookie-popup-with-overlay__button']")
     EMAIL_FORM = (By.XPATH, "//input[@placeholder='email@email.com']")
     PASSWORD_FORM = (By.XPATH, "//input[@placeholder='Password']")
     CONFIRMATION_LOG_IN_BUTTON = (By.XPATH, "//form[@data-ref='login_modal']//button[@type='submit']")
+    SIGN_IN_BUTTON = (By.XPATH, "//button[@aria-label='Log in']")
+    ACTUAL_LOG_OUT_BUTTON = (By.XPATH, "//a[@data-ref='header-dropdown-user__logout']")
+    USER_MENU = (By.XPATH, "//button[@data-ref='header-menu-item__toggle-button']/hp-header-menu-user")
+    FRAME_DATEPICKER = (By.XPATH, "//iframe[@id='_hjRemoteVarsFrame']")
 
+    @staticmethod
+    def get_search_tab(name):
+        SEARCH_TAB = (By.XPATH, f"//hp-search-widget-tab[@iconid='glyphs/{name}']")
+        return SEARCH_TAB
 
-class MainHeaderLocators:
-    SIGN_IN_BUTTON = (By.XPATH, "//button[@aria-label='Log in']")
-    DROPDOWN_PROFILE = (By.XPATH, "//logged-in")
-    LOG_OUT_BUTTON = (By.XPATH, "//ry-log-out-button//button")
 
+class SearchCarsPageLocators:
+    ACCEPT_COOKIES_BUTTON = (By.XPATH, "//button[@id='onetrust-accept-btn-handler']")
+    CAR_CARD = (By.XPATH, "//div[@data-coordinates]//div[@class='address_container']")
+    SEARCH_SUMMARY = (By.XPATH, "//table[@class='ab-SearchSummary']")
+    MAIN_LOGO = (By.XPATH, "//img[@class='rch-logo-image ']")
 
-class UserMainPageLocators:
-    INPUT_DEPARTURE_FORM = (By.XPATH, "//*[@id='input-button__departure']")
-    INPUT_DESTINATION_FORM = (By.XPATH, "//input[@id='input-button__destination']")
-    CONFIRMATION_AIRPORT = (By.XPATH, "(//span[@data-ref='airport-item__name'])[last()]")
-    FRAME_DATEPICKER = (By.XPATH, "//iframe[@id='_hjRemoteVarsFrame']")
-    SEARCH_BUTTON = (By.XPATH, "//button[@data-ref='flight-search-widget__cta']")
 
 class SearchFlightsPageLocators:
-    URL_IDENTIFIER = ("trip/flights/select")
+    URL_IDENTIFIER = "trip/flights/select"
```

Comment:
url is not a selector

Reply:
Fixed. Got out from locators to utils.url_storage